pkg/meta/redisdb/parameters.go

```diff
@@ -0,0 +1,20 @@
+package redisdb
+
+import (
+	"github.com/redis/go-redis/v9"
+)
+
+type DBDriverParameters struct {
+	URL       string
+	KeyPrefix string
+}
+
+func GetRedisClient(params DBDriverParameters) (redis.UniversalClient, error) {
+	// go-redis supports connecting via the redis uri specification (more convenient than parameter parsing)
+	opts, err := redis.ParseURL(params.URL)
```

Comment:
A suggestion, allow to use alternative [redis.UniversalClient][1], the problem of ParseURL - it supports only simple single-node servers.
But you cannot setup Sentinel or Cluster with it, which i believe, required to make HA.

[1]: https://pkg.go.dev/github.com/go-redis/redis/v9#UniversalOptions

Reply:
Was thinking the same thing. But I did not find an equivalent call returning the universal client opts from an url. My conclusion is we'd have to set opt separately in the zot configuration.

Reply:
Updated the code. Looks like there is a function to parse a Cluster url.
For Sentinel I couldn't find one, but I made progress on having individual opts in configration.